Makefile

```diff
@@ -70,6 +70,11 @@ xcodeproj-httpservermock:
 		@cd instrumented-tests/http-server-mock/ && swift package generate-xcodeproj
 		@echo "OK 👌"
 
+xcodeproj-session-replay:
+		@echo "⚙️  Generating 'DatadogSessionReplay.xcodeproj'..."
+		@cd session-replay/ && swift package generate-xcodeproj
```

Comment:
This is great! I would like to leverage this further in v2 when we start restructuring the project structure!